apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/GetSchemaTableTests.cs

```diff
@@ -32,42 +33,79 @@ public async Task GetSchemaTable_Default_ReturnsNull()
         {
             using (var connection = _fixture.GetConnection())
             {
-                var command = connection.CreateSelectCommand($"SELECT Int64Value, BytesArrayValue FROM {_fixture.TableName}");
+                var command = connection.CreateSelectCommand($"SELECT * FROM {_fixture.TableName}");
                 using (var reader = await command.ExecuteReaderAsync())
                 {
                     Assert.Null(reader.GetSchemaTable());
                 }
             }
         }
 
-        [Fact]
-        public async Task GetSchemaTable_WithFlagEnabled_ReturnsNull()
+        public static List<object[]> SchemaTestData =>
```

Comment:
Consider using `TheoryData<string, Type, SpannerDbType>`. I believe that will allow you to drop the `new object[]` everywhere. (I'd also include the `{ get } = ` so that it doesn't generate a new list each time the property is accessed.)

Reply:
Thanks for the tip, done.